examples/inverse/plot_morph_VolSourceEstimate.py

```diff
@@ -0,0 +1,282 @@
+    img = mne.save_stc_as_volume(fname, stc, src,
+                                 mri_resolution=True)
+
+    return img, stc, src
+
+
+###############################################################################
+# Save non linear mapping data
+
+def write_morph(fname, morph, overwrite=True):
+    morph_out = dict()
+
+    # dissolve object structure
+    for key, value in morph.items():
+        # save type for order independent decomposition
+        if hasattr(value, '__dict__'):
+            value = value.__dict__
+        morph_out[key] = value
+    if not fname.endswith('.h5'):
+        fname += '.h5'
+    write_hdf5(fname, morph_out, overwrite=overwrite)
+
+
+###############################################################################
+# Load non linear mapping data
+
+def read_morph(fname):
+    if not fname.endswith('.h5'):
+        fname += '.h5'
+    morph_in = read_hdf5(fname)
+
+    morph = dict()
+    # create new instances
+    morph['mapping'] = imwarp.DiffeomorphicMap(None, [])
+    morph['mapping'].__dict__ = morph_in.get('mapping')
+    morph['affine'] = imaffine.AffineMap(None)
+    morph['affine'].__dict__ = morph_in.get('affine')
+
+    morph['affine_reg'] = morph_in.get('affine_reg')
+    morph['domain_shape'] = morph_in.get('domain_shape')
+
+    return morph
+
+
+###############################################################################
+# Execute example
+
+# Settings
+voxel_size = (5., 5., 5.)  # of the destination volume
+
+# Setup path
+data_path = sample.data_path()
+
+t1_sample = data_path + '/subjects/sample/mri/brain.mgz'
+t1_fsaverage = data_path + '/subjects/fsaverage/mri/brain.mgz'
+
+# compute LCMV beamformer inverse example
+img, stc, src = compute_lcmv_example_data(data_path)
+
+###############################################################################
+# Compute Morph Matrix and Dispersion Volume
+morph = mne.source_estimate.compute_morph_sdr(
+    t1_sample,
+    t1_fsaverage,
+    voxel_size=voxel_size)
+
+write_morph('SDR-sample-fsaverage', morph)
+
+###############################################################################
+# Morph Pre - Computed
+
+morph_precomputed = read_morph('SDR-sample-fsaverage')
+
+stc_vol_to_preC = stc.morph_precomputed('sample', 'fsaverage', src,
+                                        morph_precomputed,
+                                        as_volume=False)
+
+img_vol_to_preC = stc.morph_precomputed('sample', 'fsaverage', src,
+                                        morph_precomputed,
+                                        as_volume=True)
+
+###############################################################################
+# Morph
+stc_vol_to_drct = stc.morph('sample',
+                            'fsaverage',
+                            t1_sample,
+                            t1_fsaverage,
+                            src,
+                            voxel_size=voxel_size,
+                            as_volume=False)
```

Comment:
This is mostly redundant. Since `vol` is so slow, perhaps we should only have the `precomputed` method for now. (I wish we hadn't provided `morph` and `morph_precomputed` for surfaces in the first place... precomputed is almost always faster.)

Reply:
I mean, since I'm most likely going to restructure some parts anyway, we could just keep morph for now, but change it such that it internally computes ``compute_morph`` and ``morph_precomputed``